setup.py

```diff
@@ -0,0 +1,64 @@
+"""A setuptools based setup module for pm-utils"""
+
+
+from os import path
+# Always prefer setuptools over distutils
+from setuptools import setup, find_packages
+
+HERE = path.abspath(path.dirname(__file__))
+NAME = "pm_utils"
+
+# Load the version
+with open(path.join(HERE, NAME, "version.py")) as version_file:
+    exec(version_file.read())
+
+# Get the long description from the README file
+with open(path.join(HERE, "README.md"), encoding="utf-8") as f:
+    LONG_DESCRIPTION = f.read()
+
+REQUIREMENTS = [
+    "pandas",
+    "xlrd",
+    "openpyxl",
+]
+
+# TODO
+TEST_REQUIREMENTS = [
+]
+
+# TODO
+SETUP_REQUIREMENTS = [
+]
+
+setup(
+    name=NAME,
+    version=__version__,
+    description="Utilities for project managers",
+    long_description=LONG_DESCRIPTION,
+    long_description_type= "text/markdown",
+    # applicable after transferring this repo to BCM-HGSC private
+    author="Baylor College of Medicine - Human Genome Sequencing Center",
+    author_email="questions@hgsc.bcm.edu",
+    classifiers=[
+        "Development Status :: 3 - Alpha",
+        "Environment :: Console",
+        "Intended Audience :: Developers",
+        "Natural Language :: English",
+        "License :: OSI Approved :: MIT License",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+    ],
+    packages=find_packages(exclude=["contrib", "docs", "tests"]),
+    install_requires=REQUIREMENTS,
+    tests_requires=TEST_REQUIREMENTS,
```

Comment:
The `tests_requires` doesn't really get used, so you don't get what you want. Right now, that doesn't matter, but you would need to manually install `pytest` if you plan on using such tests.

If you aren't planning on using `pytest`, just drop the `tests_requires=TEST_REQUIREMENTS,` line and the paragraph above that defines `TEST_REQUIREMENTS`.

Later, if you want to add support for `pytest` add this code:

```python
TEST_REQUIREMENTS = [
    "pytest",
]

EXTRAS_REQUIRE = {  # Optional
    'test': TEST_REQUIREMENTS,
}

...
setup(
    ...
    extras_require=EXTRAS_REQUIRE,
    ...
)
```

See this [example](/BCM-HGSC/bhaou_catapult/blob/32bc6ee369da3f809b23dd257f4606b33a78a7c9/setup.py#L48)

Reply:
That is my next item to do.  I plan to create another branch for issue #11 and add tests.

Reply:
Then I suggest deleting line 51, since it doesn't do anything useful.

Reply:
True.  Removing the unnecessary requirements.